### ISSUE TITLE
comma-spacing should handle parentheses

### ISSUE THREAD

**Issue by btmills** [eslint/eslint#1506]
On latest `master`:

``` js
// test.js
(a = b), c;
[(a * b), c];
```

``` sh
$ eslint --reset --no-eslintrc --rule "comma-spacing: 2" test.js

test.js
  2:9   error  There should be no space before ','  comma-spacing
  3:10  error  There should be no space before ','  comma-spacing

✖ 2 problems
```

Both errors are incorrect.
